docs/user/exams/students_guide.rst

```diff
@@ -117,7 +120,7 @@ Exam Conduction
 - You can also navigate through the exercises when you are done with one by clicking |save_continue|. This action will save and submit your changes and move to the next exercise.
 
     .. warning::
-        For programming exercises, you must manually press |submit| otherwise your solution will **not** be counted!
+        For programming exercises, there is no save button. You must manually press |submit| otherwise your solution will **not** be counted!
```

Comment:
```suggestion
        For programming exercises, there is no save button. You must manually press |submit| otherwise, your solution will **not** be graded!
```